src/Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -38,7 +38,7 @@
         <a href="#">
             {%- set translation_domain = item.extra('label_catalogue', 'messages') -%}
             {%- set icon = item.extra('icon')|default('') -%}
-            {{ icon|raw }}
+            {{ icon|parse_icon|raw }}
```

Comment:
Since the icon value can be `''`, an early check should be added to the parse_icon method in order to return `null` if no icon is passed.